[rocprofiler-sdk] Update SPM docs - minimum required driver version (#7800)
ROCm/rocm-systems: projects/rocprofiler-sdk/source/docs/how-to/using-spm.rst

      SPM                 :   Supported
      Dimensions          :   DIMENSION_INSTANCE[0:15] DIMENSION_XCC[0:7]

The preceding output shows that the TCC_MISS counter can be sampled.

.. note::

>    SPM requires AMD GPU Driver version **6.19.14.31400000** or newer to work correctly.

Reviewer: ```suggestion
   For proper functioning, SPM requires AMD GPU Driver version **6.19.14.31400000** or later.
```

>    Verify the loaded ``amdgpu`` kernel module version before using SPM.

Reviewer: ```suggestion
   Before using SPM, verify the loaded ``amdgpu`` kernel module version.
```

To check the driver version, use:

.. code-block:: bash

  cat /sys/module/amdgpu/version

  # Example output:
  # 6.19.14.31400000

> The value is also reported by ``amd-smi version`` on DKMS-built systems.

Reviewer: ```suggestion
You can also check this driver version using ``amd-smi version`` command on DKMS-built systems.
```
